cmd/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	cobbler "github.com/ContainerSolutions/cobblerclient"
+)
+
+var config = cobbler.ClientConfig{
```

Comment:
Maybe it's a good idea to use the `flag` package here to make all these variables overridable. Can't test in my local env. without modifying the source.